R/bs4_book.R

```diff
@@ -21,12 +21,12 @@
 #'   regardless of the size of your device. A sticky header gets out of your
 #'   way when reading, but is easily accessible if you need it.
 #'
-#' * In-line footnotes mean you can read asides next to the next they refer
-#'   to. This theme is best paired with a reference style that generates
-#'   footnotes.
+#' * In-line footnotes mean you can read asides next to the text they refer
+#'   to. If you use a bibliography, this bookdown theme is best paired with 
+#'   a reference style (CSL file) that generates footnotes.
```

Comment:
it'd be nice to explain where one would put a reference to the CSL file (is it in index.Rmd?)

Reply:
Yeah, needs something like the instructions at https://hugodown.r-lib.org, i.e. download a footnote style csl-file (e.g. chicago-fullnote-bibliography.csl) put this in your `index.Rmd`:

```yaml
bibliography: refs.bib
csl: chicago-fullnote-bibliography.csl
```

And then optionally, if you no longer want a reference section at the back of the book:

```yaml
suppress-bibliography: true
```

Reply:
I added that and in the end while I was at it I drafted something about customization.